src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString.split(';');
+  const cssObject = {};
+  let objectEntries = [];
+
+  objectEntries = cssProperties.map(element => (element.split(':')));
```

Comment:
```suggestion
  let objectEntries = cssProperties.map(element => (element.split(':')));
```
Навіщо створювати пустий массив, а потім його преезаписувати на новий?